requirements.txt

```diff
@@ -9,14 +9,13 @@ rpy2==3.0.4
 rpy2-bioconductor-extensions==0.2.5.dev0
 pysam==0.15.3
 notebook==6.4.1
-mofapy==1.2
 jupyter==1.0.0
 igv==0.9.6
 gsheets==0.3.2
 gseapy==0.9.13
 google-api-core==1.10.0
 google-cloud-storage==1.15.0
 goatools==0.9.5
-firecloud-dalmatian==0.0.10
+firecloud-dalmatian
```

Comment:
we need to precise the version for this one. anything below 0.0.10 would break

Reply:
resolved in new PR